serverless/knative_eventing/serverless-channels.adoc

```diff
@@ -0,0 +1,29 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-channels"]
+= Using Channels
+include::modules/common-attributes.adoc[]
+:context: serverless-channels
+
+toc::[]
+
+It is possible to sink events from an event source to a Knative Eventing channel.
+Channels are Kubernetes Custom Resources (CRs) which define a single event forwarding and persistence layer.
+After events have been sent to a channel, these events can be sent to multiple Knative services by using a subscription.
+
+The default configuration for channel instances is defined in a ConfigMap by the cluster administrator, however developers can still create their own channels directly by instantiating a supported channel object.
+
+== Supported channel types
+Currently, {ServerlessProductName} only supports the use of InMemoryChannel channels as part of the Knative Eventing Technology Preview.
+
+[IMPORTANT]
+====
+InMemoryChannel channels are a development only channel and are not suitable for use in production.
+====
+
+The following are limitations of InMemoryChannel channels:
+
+* No event persistence is available. If a pod is goes down, events on that pod will be lost.
+* InMemoryChannel channels do not implement event ordering, so two events that are received in the channel at the same time may be delivered to a subscriber in any order.
+* If a subscriber rejects an event, there are no redelivery attempts. Instead, the rejected event will be sent to a dead letter sink if this exists, or is otherwise dropped.
```

Comment:
Do we document how to configure a dead letter sink somewhere?

Reply:
Nope but I can add it - I'll track it under a separate PR though to prevent blocking this one further.

Reply:
Now tracked in JIRA.